backend/app/main/answer_views.py

```diff
@@ -20,12 +24,68 @@ def answer(question_id):
             question=question,
             session_id=session['id']))
 
+    
+    #Store it in the Airtable
+    if question_id == '1':
+        field_name = "Name"
+    else:
+        field_name = "Email"
+    
+    temp_info = {
+        field_name: request.values['Body'],
+    }
+    temp_field = {"fields": temp_info}
+    temp_json = {"records": [temp_field]}
+
+   # headers = {'Authorization': 'Bearer keybp9BmMeD5OLIjq', 'Content-Type': 'application/json; charset=utf-8'}
+    req = requests.post('https://api.airtable.com/v0/appw4RRMDig1g2PFI/SMS%20Responses',json=temp_json,headers={"Authorization": str(os.environ.get("API_KEY"))})
+    response_json = req.json()
+    
+    for r in response_json["records"]:
+        row_id = r["id"]
+
     next_question = question.next()
     if next_question:
         return redirect_twiml(next_question)
     else:
         return goodbye_twiml()
 
+#My own route
```

Comment:
(nitpicking) Put a space after comments just so that they all have the same style: `# My own route`